public/js/script.js

```diff
@@ -32,6 +35,7 @@ function updateSelector(screen_data) {
     }
 }
 
+// FIXME does not corectly switch between screens
```

Comment:
[ROTFL](http://www.bonkersworld.net/images/2012.08.15_todo.png)

(plus you made a spelling mistake in your fixme comment! LOL)